content/statefulset/statefulset.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Create StatefulSet"
+date: 2018-08-07T08:30:11-07:00
+weight: 20
+---
+#### Introduction
+StatefulSet consists of serviceName, replicas, template and volumeClaimTemplates.
+**serviceName** is "mysql", headless service we created in previous section, **replicas** is 3, the desired number of pod, **template** is the configuration of pod, **volumeClaimTemplates** is to claim volume for pod based on storageClassName, gp2 that we created in "Define Storageclass" section. [Percona Xtrabackup](https://www.percona.com/software/mysql-database/percona-xtrabackup) is in template to clone source MySQL server to its slaves.
+#### Create StatefulSet
+Copy/Paste the following commands into your Cloud9 Terminal.
+```
+cd ~/environment/templates
+wget https://eksworkshop.com/statefulset/statefulset.files/mysql-statefulset.yml
+```
+Check statefulset "mysql" created by following command.
+```
+kubectl create -f ~/environment/templates/mysql-statefulset.yml
+statefulset "mysql" created
+```
+#### Watch StatefulSet
+Watch the status of statefulset.
+```
+kubectl get -w statefulset
+NAME      DESIRED   CURRENT   AGE
```

Comment:
If you want to provide example output, break it into a separate code box so it doesn't get copy/pasted with the cat command. There are examples elsewhere of wording like "The output should look something like this"